src/modules/hotkeys/main.ts

```diff
@@ -113,6 +136,19 @@ export const registerHotkeys = async (
             }
         }
         if (callback) {
+            // disable native breadcrumb hotkey
+            if (hotkey.split(' ').includes('b')) window.breadcrumbnav.clear();
```

Comment:
I think it would be better if we'll define the hotkey next to the other ones.

Reply:
unfortunately not possible due to the system ho the breadcrumb-hotkey is implemented in comparison to the other native hotkeys in missions window.

Reply:
Ok, that's a good point. But I still think we should move it elsewhere, because it's difficult to find there. At least for me in 6 months :D

Reply:
any ideas / suggestions?
I did put it there because I only want to disable the native one if there's any custom hotkey that uses `b` as a standalone key.

Reply:
Maybe as const in the main.ts on top?